src/wasm/wasm-ir-builder.cpp

```diff
@@ -891,8 +888,8 @@ Result<> IRBuilder::visitCatch(Name tag) {
     delimiterLocs[delimiterLocs.size()] = lastBinaryPos - codeSectionOffset;
   }
 
-  pushScope(
-    ScopeCtx::makeCatch(tryy, originalLabel, label, labelUsed, branchLabel));
```

Comment:
Was it a bug we didn't CHECK_ERR here before?

Reply:
No, `pushScope` did not previously return a `Result<>`.